src/commands/helm-lint.yaml

```diff
@@ -0,0 +1,8 @@
+---
+parameters:
+  chart:
+    type: "string"
+description: Lint Helm chart
```

Comment:
```suggestion
description: architect/helm-lint: Lint Helm chart
```

Reply:
That breaks the build.

Reply:
Should be `description: "architect/helm-lint: Lint Helm chart"`